scripts/qa_crowbarsetup.sh

```diff
@@ -4211,6 +4211,7 @@ function onadmin_addupdaterepo
         onadmin_setup_local_zypper_repositories
         ensure_packages_installed createrepo
         createrepo -o $UPR $UPR || exit 8
+        gpg --detach-sign --armor --yes $UPR/repodata/repomd.xml
```

Comment:
It seems, this is part of the crowbar package outside of git:
https://build.opensuse.org/package/show/systemsmanagement:crowbar:4.0/crowbar